drivers/i2c/i2c_mcux_flexcomm.c

```diff
@@ -292,6 +346,29 @@ static void i2c_target_transfer_callback(I2C_Type *base,
 	}
 }
 
+static int mcux_flexcomm_setup_slave_config(const struct device *dev)
```

Comment:
NIT: The diff would be a bit easier to read if you added this helper in a separate commit before adding the multi-slave support

Reply:
Moved to separate commit